timemachine/potentials/bonded.py

```diff
@@ -147,6 +148,88 @@ def harmonic_bond(conf, params, box, lamb, bond_idxs, lamb_mult=None, lamb_offse
     return jnp.sum(energy)
 
 
+def get_cos_angle(ci, cj, ck):
+
+    ci = jnp.array(ci)
+    cj = jnp.array(cj)
+    ck = jnp.array(ck)
+
+    # compute cosine angle of three points
+    vij = ci - cj
+    vjk = ck - cj
+    top = jnp.sum(jnp.multiply(vij, vjk), -1)
+    bot = jnp.linalg.norm(vij, axis=-1) * jnp.linalg.norm(vjk, axis=-1)
```

Comment:
Naming nit: subscript "ij" on L158 uses convention "first minus second", subscript "jk" on L159 uses convention "second minus first" -- prefer to standardize one way or the other within the definition

```suggestion
    # compute cosine angle of three points
    vij = ci - cj
    vkj = ck - cj
    top = jnp.sum(jnp.multiply(vij, vkj), -1)
    bot = jnp.linalg.norm(vij, axis=-1) * jnp.linalg.norm(vkj, axis=-1)
```

Reply:
sgtm!